release-notes/v5.6.1.md

```diff
@@ -1,3 +1,7 @@
+#### <sub><sup><a name="v561-4421" href="#v561-note-git-resource-273">:link:</a></sup></sub> feature
```

Comment:
to avoid the scenario where clicking one release note scrolls you to another
```suggestion
#### <sub><sup><a name="v561-4421" href="#v561-4421">:link:</a></sup></sub> feature
```